bin/gftools-builder.py

```diff
@@ -72,12 +80,33 @@
     builder.config["logLevel"] = "DEBUG"
 
 if args.dump_config:
-    import sys
     import yaml
 
     with open(args.dump_config, "w") as fp:
         config= {k: v for (k, v) in builder.config.items() if v is not None}
         fp.write(yaml.dump(config, Dumper=yaml.SafeDumper))
     sys.exit()
 
+try:
```

Comment:
We're checking the yaml after the builder has been initialised (see L64) when we actually want to check it during initialisation. What we want to do is replace current yaml loading routine in `Lib/gftools/builder/__init__.py L112` with your code below. I'd also wrap this code into its own helper function e.g

```PYTHON
def load_builder_config(fp, schema=schema):
    ....
```

Reply:
Thanks. This is done in commit 9463326.